File: Python/BinaryTree/ConstructingBinaryTree.py

# @author
# Aakash Verma

# Creating a structure for the node.
# Initializing the node's data upon calling its constructor.
class Node:
    def __init__(self, data):
        self.data = data
        self.right = self.left = None


# Defining class for the Binary Tree
class BinaryTree:
    
    # Assigning root as null initially. So as soon as the object will be created 
    # of this BinaryTreeTraversal class, root will be set as null.
    def __init__(self): 
        self.root = None

# main method
if __name__ == '__main__':
    tree = BinaryTree()
    tree.root = Node(1)
    tree.root.left = Node(2)
    tree.root.right = Node(3)
    tree.root.left.left = Node(4)
    tree.root.left.right = Node(5)
    tree.root.right.left = Node(6)
    tree.root.right.right = Node(7)

File: Java/BinaryTree/ConstructingBinaryTree.java
/*
*
*   @author
*   Aakash Verma
*
*/


/*  
    Creating a structure for the node.
    Initializing the node's data upon calling its constructor.
*/
class Node {
    int data;
    Node left, right;
    
    public Node(int key) {
        data = key;
        left = right = null;
    }
}


class BinaryTree {

	/* Creating a root node for the Tree */
    static Node root;
    
    /* 
        Assigning root as null initially. So as soon as the object will be created 
        of this BinaryTreeTraversal class, root will be set as null.
    */
    BinaryTree() {
        root = null;
    }

	/* main method */
    public static void main (String[] args) {
        BinaryTree tree = new BinaryTree();
        tree.root = new Node(1);
        tree.root.left = new Node(2);
        tree.root.right = new Node(3);
        tree.root.left.left = new Node(4);
        tree.root.left.right = new Node(5);
        tree.root.right.left = new Node(6);
        tree.root.right.right = new Node(7);
}